sae_lens/wandb_compat.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+import os
+import uuid
+from typing import Any
+
+
+def _load_backend() -> tuple[Any, str]:
+    backend = os.getenv("SAE_LENS_LOGGING_BACKEND", "auto").lower()
+    if backend not in {"auto", "wandb", "swanlab"}:
+        backend = "auto"
+
+    if backend in {"auto", "swanlab"}:
```

Comment:
I'd rather the default be wandb, not swanlab. If a user just happens to have SAELens and swanlab deps in the same project their logging will seem to break and this will be very hard to debug. IMO if users want to use swanlab they should need to opt-in specifically with a ENV var.